docs/setup/settings.asciidoc

```diff
@@ -249,77 +272,73 @@ To reload the logging settings, send a SIGHUP signal to {kib}.
 [cols="2*<"]
 |===
 
-|[[logging-dest]] `logging.dest:`
- | Enables you to specify a file where {kib} stores log output.
-*Default: `stdout`*
-
-| `logging.json:`
- | Logs output as JSON. When set to `true`, the logs are formatted as JSON
-strings that include timestamp, log level, context, message text, and any other
-metadata that may be associated with the log message.
-When <<logging-dest, `logging.dest.stdout`>> is set, and there is no interactive terminal ("TTY"),
-this setting defaults to `true`. *Default: `false`*
+|[[logging-root]] `logging.root:`
```

Comment:
Should we include link(s) in this section that link to the more advanced documentation?

Reply:
Links will be added in a follow up PR. We can't link to docs we're adding within the same PR.